sqlmesh/core/engine_adapter/base.py

```diff
@@ -407,6 +409,24 @@ def create_index(
         )
         self.execute(expression)
 
+    @staticmethod
```

Comment:
[Nit] What's the rationale behind using `@staticmethod` here? Should we just make this a normal function helper?